### PR TITLE
Add restore Color Button

### PR DESCRIPTION
**Pull request by sukritsangvong**
Quality-of-life feature (I want to get the default color back)

image

A bunch of formatting, and removed a console.log()

### CODE REVIEW

File: wikichange/assets/options.css [hunk 1]
```diff
@@ -1,45 +1,49 @@
...
-  row-gap: 7%;
-  align-items: center;
-  justify-items: center;
-
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    grid-template-areas:
+        "label input"
+        "restore restore"
```
Comment (sukritsangvong) [inline wikichange/assets/options.css:10]: added this

File: wikichange/src/options/options.html [hunk 2]
```diff
@@ -16,7 +16,7 @@ <h1>Options</h1>
     <div>
         <input name="highlight-color" id="highlight-color" type="color" />
     </div>
-
+    <input type="button" value="Restore Color" id="restore-color"/>
```
Comment (sukritsangvong) [inline wikichange/src/options/options.html:19]: added this

File: wikichange/assets/options.css [hunk 3]
```diff
@@ -1,45 +1,49 @@
...
-  grid-area: submit;
+    grid-area: submit;
+
+    max-width: min(15%, 150px);
+}
 
-  max-width: min(15%, 150px);
+#restore-color {
+    grid-area: restore;
 }
```
Comment (sukritsangvong) [inline wikichange/assets/options.css:42]: added this

File: wikichange/src/options/options.js [hunk 4]
```diff
@@ -2,34 +2,42 @@
...
+document.addEventListener("DOMContentLoaded", restore_options);
+document.querySelector("form").addEventListener("submit", (event) => {
     event.preventDefault();
     save_options();
-})
\ No newline at end of file
+});
+document.getElementById("restore-color").addEventListener("click", () => {
+    document.getElementById("highlight-color").value = "#AFE1AF";
+});
```
Comment (sukritsangvong) [inline wikichange/src/options/options.js:43]: added this

File: wikichange/src/options/options.js [hunk 5]
```diff
@@ -2,34 +2,42 @@
...
-        document.getElementById('highlight-color').value = items.highlightColor;
-    });
+    chrome.storage.sync.get(
+        {
+            highlightColor: "#AFE1AF",
+        },
+        function (items) {
+            document.getElementById("highlight-color").value = items.highlightColor;
+        }
+    );
```
Comment (sukritsangvong) [inline wikichange/src/options/options.js:33]: Not sure what this is doing, but the color never changes back to the default highlight for me. @RichartE

File: wikichange/src/options/options.js [hunk 6]
```diff
@@ -2,34 +2,42 @@
...
 
 /** Restores select box and checkbox state using the preferences
  * stored in chrome.storage.
  */
 function restore_options() {
     /** Use default value highlightColor #AFE1AF */
-    chrome.storage.sync.get({
-        highlightColor: '#AFE1AF'
-    }, function (items) {
-        console.log(items);
```
Comment (sukritsangvong) [inline wikichange/src/options/options.js:26]: removed this

**Review by LuisaE — APPROVED** [review]
Thank you!

**Pull request merged**